StringCaculatorTests/StringCaculatorTests.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  StringCaculatorTests.swift
+//  StringCaculatorTests
+//
+//  Created by jinho jeong on 2022/04/25.
+//
+
+import XCTest
+@testable import RacingCar
+
+class StringCaculatorTests: XCTestCase {
+    
+    var sut: StringCaculator!
+    
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        sut = StringCaculator()
+    }
+
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        sut = nil
+    }
+
+    func test_splitString호출시_1_2을넣었을때_1_2배열이_나오는지테스트() {
```

Comment:
```suggestion
    func test_splitString_호출시_1_2을넣었을때_1_2배열이_나오는지_테스트() {
```

영어와 한글을 섞어 쓴 이유가 있을까요?.? 하나로 통일하는게 좀 더 일관성 있지 않을까 생각합니다.
(이건 제 의견이라서 한 번 생각만 해봐주셔도 좋을거 같아요 😄 )